cargo-quickinstall/src/json_value_ext.rs

```diff
@@ -27,16 +27,16 @@ impl JsonExtError {
 }
 
 pub trait JsonValueExt {
-    fn get_owned(self, key: &dyn JsonKey) -> Result<JsonValue, JsonExtError>;
+    fn extract_from_value(self, key: impl JsonKey) -> Result<JsonValue, JsonExtError>;
```

Comment:
in my previous comment, I was complaining about JsonKey::get_owned. Please keep JsonValueExt::get_owned how it was

Reply:
Revertion done.